saleor/core/tracing.py

```diff
@@ -40,9 +40,9 @@ def webhooks_otel_trace(
         kind=SpanKind.CLIENT,
         links=span_links,
     ) as span:
-        if app:
-            span.set_attribute(saleor_attributes.SALEOR_APP_ID, app.id)
-            span.set_attribute(saleor_attributes.SALEOR_APP_NAME, app.name)
+        span.set_attribute(saleor_attributes.SALEOR_APP_ID, app.id)
+        span.set_attribute(saleor_attributes.SALEOR_APP_IDENTIFIER, app.identifier)
+        span.set_attribute(saleor_attributes.SALEOR_APP_NAME, app.name)
```

Comment:
`SALEOR_APP_IDENTIFIER` added to be consistent with webhooks metrics.
Legacy attributes are kept for backwards compatibility.